tests/07-remove.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+test_description='Test the remove command'
+
+WHEREAMI=$(dirname "$(realpath "$0")")
+. $WHEREAMI/setup.sh
+
+test_expect_success 'cabin remove tbb mydep toml11' '
+    mkdir remove_test &&
+    cp "$WHEREAMI/../cabin.toml" remove_test/cabin.toml &&
+
+    test_when_finished "rm -rf remove_test" &&
+
+    (
+        cd remove_test &&
```

Comment:
The common style is:
```suggestion
    cd remove_test &&
    (
```

And I would put all the following lines within the paren.